bin/kn

```diff
@@ -191,6 +204,9 @@ kubetoken)
 # KubeNow-defined commands that need PWD validation
 apply | destroy | provision | scale | kubectl | helm | ssh)
   validate_pwd
+  # get provisioner image from tfvar-file
+  read_provisioner_image_var
```

Comment:
Have provisioners_img-var to config file (to allow different configs have different version of provisioner image/version tag)